### PR TITLE
Revert "Hypospray sprite change and some tweaks"

### PR DESCRIPTION
**Pull request by furrycactus**
This change from wezzy didn't go through a feedback process and replaced the brand new and widely approved sprites for hyposprays that were made by BRAINOS. Aside from the fact that community opinion wasn't gathered on this sprite change, it just seemed awfully rude and disrespectful to come in and totally replace the prior, barely month old sprites from BRAINOS, and then state on the forums that he refuses to accept a full revert of his own work when people objected.

So, I'm arguing that it _should_ be fully reverted until feedback is gathered.

Forum thread in question.
https://forums.aurorastation.org/topic/13882-the-great-hypospray-debate/

### CODE REVIEW

File: code/modules/reagents/reagent_containers/hypospray.dm [hunk 1]
```diff
@@ -4,9 +4,7 @@
 
 /obj/item/reagent_containers/hypospray
 	name = "hypospray"
-	desc = "A sterile, air-needle autoinjector for administration of drugs to patients."
-	description_fluff = "The Zeng-Hu Pharmaceuticals' Hypospray - 9 out of 10 doctors recommend it!"
-	description_info = "Unlike a syringe, reagents have to be poured into the hypospray before it can be used."
```
Comment (Geevies) [inline code/modules/reagents/reagent_containers/hypospray.dm]: keep this description_info in

File: code/modules/reagents/reagent_containers/hypospray.dm [hunk 2]
```diff
@@ -32,37 +31,38 @@
 
 /obj/item/reagent_containers/hypospray/cmo
 	name = "premium hypospray"
-	desc = "A high-end version of the regular hypospray, it allows for a substantially higher rate of drug administration to patients."
-	description_fluff = "The Zeng-Hu Pharmaceuticals' Hypospray Mk-II is a cutting-edge version of the regular hypospray, with a much more expensive and streamlined injection process."
-	description_info = "This version of the hypospray has no delay before injecting a patient with reagent."
```
Comment (Geevies) [inline code/modules/reagents/reagent_containers/hypospray.dm]: this one too

File: code/modules/reagents/reagent_containers/hypospray.dm [hunk 3]
```diff
@@ -32,37 +31,38 @@
...
+	desc = "The Zeng-Hu Pharmaceuticalsn premium hypospray is a cutting-edge, sterile, air-needle autoinjector for rapid administration of drugs to patients."
+	item_state = "cmo_hypo"
 	volume = 30
 	time = 0
 
 /obj/item/reagent_containers/hypospray/attack(var/mob/M, var/mob/user, target_zone)
 	. = ..()
 	var/mob/living/carbon/human/H = M
 	if(istype(H))
-		user.visible_message(span("warning", "\The [user] is trying to inject \the [M] with \the [src]!"),span("notice", "You are trying to inject \the [M] with \the [src]."))
```
Comment (Geevies) [inline code/modules/reagents/reagent_containers/hypospray.dm]: keep these

**Comment by fernerr** [comment]
As the tweaks were supposedly made with Brainos' blessing, as stated in the previous pr, I'm not sure how the 1 month revert rule applies here.

**Comment by furrycactus** [comment]
!review

**Comment by natiiform** [comment]
the changes were not with brainos’s blessing and i do not know where the poster of the previous pr got that idea. just throwing that out there

**Comment by alsoandanswer** [comment]
I think it would
Hspray
 be more fair to revert to the older sprite of the hypospray.

**Comment by sexypompadour** [comment]
christ no, i don't care whose sprites go in at this point, as long as we get rid of those fuckin welder looking things

like, if your whole critique of mine was that they looked like they'd get sent to research (which is totally understandable!) then why would you want to take two steps backwards and have those welder looking things that greytide cargo techs will send to engineering? those old sprites are hideous. do not bring them back.

**Comment by alsoandanswer** [comment]
I would be happy to oblige by using an alternative sprite from guns.dmi or something.

**Comment by resilynn** [comment]
I do not think a compromise is needed or warranted, the community feedback was very clear.

**Comment by furrycactus** [comment]
Far more fair is reverting to the sprite that was already accepted. Feedback is not necessary, as it has _already_ been gathered, both in a prior thread, and yet again in the current one with a majority of people desiring the Zeng-Hu sprites back.

**Comment by furrycactus** [comment]
That's out of the scope for this PR. This PR's only purpose is to revert back to previously accepted sprites until new ones get feedback and approval, or to revert back if majority approval goes in favour of BRAINOS's ones. I'm not here to fix sprites or make up new sprites and add them, unless someone's going to drop a .dmi file in my lap to stick in.

**Comment by furrycactus** [comment]
> revert back to previously accepted sprites until new ones get feedback and approval

Please read the whole thing.

**Comment by sexypompadour** [comment]
yo i'll just tweak em to account for the critique, since it's valid and i can't unsee it.

can y'all wait like two or 3 days?

**Comment by furrycactus** [comment]
Definitely, yeah. I'll set DNM for now.

**Comment by furrycactus** [comment]
Closing this.

**Pull request closed without merging**